docs/docs/rest-api/public/api/v2/types/group.raml

```diff
@@ -100,3 +65,16 @@ types:
         minimum: 0
       version?:
         type: datetime
+      enforceRole?:
+        type: EnforceRole
+  GroupPartialUpdate:
+    type: object
+    properties:
+      id: strings.PathId
```

Comment:
I really don't know if this should be part of the update and nor that path.

Reply:
What do you mean exactly? I think it could be a different/dedicated endpoint if you mean that.

Reply:
I think the update should be on `:8080/v2/groups/dev` but not on `:8080/v2/groups/dev/jenkins`. And id is not required in the body.